components/layout/footer/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, Typography } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import { HeartSVG, IPXSVG } from '@/components/svg';
+
+import SocialSection from './socials-section';
+
+const Footer: FC = () => (
+  <Box
+    fontSize="s"
+    display="flex"
+    flexDirection="column"
+    textTransform="uppercase"
+  >
+    <Box textAlign="center" mb="l">
```

Comment:
This box is like a div and if you want to align elements inside them use display